examples/modular-drivers/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
-name = "micro-rdk-esp32-sensor-examples"
-resolver = "2"
-version = "0.2.0"
-edition = "2021"
-authors = ["Viam, inc. <contact@viam.com>"]
-resolver = "2"
-rust-version = "1.75"
-license = "AGPL-3.0"
+name = "modular-drivers"
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
 
 [dependencies]
-log = "0.4"
-micro-rdk = {version = "0.2.0", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32"], rev = "ad31cf1"}
+log.workspace = true
+micro-rdk = { workspace = true, features = ["esp32"] }
```

Comment:
Two of the sensors are esp32 specific (heap, wifi) and the other two are not. So, this crate needs to have an esp32 feature, and it needs to interact with the micro-rdk feature flag for esp32 support.

Reply:
I think this shouldn't be unconditionally enabling the `esp32` feature in `micro-rdk`, per above.